Updated README with basic usage information (#8)
vtex-apps/react-portal: README.md

# React Portal

# Basic Usage

> It is possible to wrap a React element with [Overlay](https://github.com/vtex-apps/react-portal/blob/master/react/Overlay.tsx) to make use of its advantages such as making the block overlap any other block dispite the its z-index

Reviewer: ```suggestion
It is possible to wrap a React element with [Overlay](https://github.com/vtex-apps/react-portal/blob/master/react/Overlay.tsx) to make use of its advantages such as making the block overlap any other block despite its z-index.
```

# React Portal

# Basic Usage

> It is possible to wrap a HTML block with [Overlay](https://github.com/vtex-apps/react-portal/blob/master/react/Overlay.tsx) to make use of its advantages such as making the block overlap any other block dispite the its z-index

Reviewer: ```suggestion
It is possible to wrap a React element with [Overlay](https://github.com/vtex-apps/react-portal/blob/master/react/Overlay.tsx) to make use of its advantages such as making the element overlap any other block despite the its z-index
```

## Example:

```
<Overlay>

>   <div className={boxClasses} style={boxPositionStyle} ref={boxRef}>

Reviewer: Change it to a more simple example
```suggestion
  <div>Hello world</div>
```
